testkit/src/main/scala/org/bitcoins/testkit/core/gen/PSBTGenerators.scala

```diff
@@ -0,0 +1,119 @@
+package org.bitcoins.testkit.core.gen
+
+import org.bitcoins.core.config.BitcoinNetwork
+import org.bitcoins.core.protocol.script.ScriptPubKey
+import org.bitcoins.core.protocol.transaction.TransactionOutput
+import org.bitcoins.core.psbt.{
+  GlobalPSBTMap,
+  GlobalPSBTRecord,
+  InputPSBTMap,
+  OutputPSBTMap,
+  PSBT
+}
+import org.bitcoins.core.wallet.builder.BitcoinTxBuilder
+import org.bitcoins.core.wallet.fee.FeeUnit
+import org.bitcoins.core.wallet.utxo.BitcoinUTXOSpendingInfo
+import org.scalacheck.Gen
+
+import scala.concurrent.{ExecutionContext, Future}
+
+object PSBTGenerators {
```

Comment:
It'd be useful to have generators for other things like `Version`, `XPubKey`, `BIP32DerivationPath`, and `Unknown`s

Reply:
Agreed, though since the use of such generators will probably depend on how they are needed (i.e. they will likely be dependent, as opposed to independent, generators which will either take other generators as inputs or will be fed to other generators), I will leave their implementation to someone who sees more clearly what their use should be